src/driver.ts

```diff
@@ -40,8 +40,25 @@ export interface Device {
     write: (values: number[] | Buffer) => number;
   };
   type: DeviceType;
+  serialNumber?: string;
```

Comment:
My guess is that we can guarantee that there'll be a serial number for a Litra device (i.e. a device with the expected vendor ID and product ID) - even if `node-hid` thinks it could be missing. What was your thinking behind the `?` here?

Reply:
`node-hid` has it as optional, so I mapped that through. However, we could set it as non-optional and instead set it as `null` if `node-hid` doesn't have a value for it?

Reply:
Let's see it as non-optional. What do you think of forcing Typescript to treat it as a string? I think we can be pretty confident that all of these devices are going to have a serial number. You can just add `as string` when you use it to do that.

Reply:
Ok, will go with `as string`.

Reply:
Updated in https://github.com/timrogers/litra/pull/86/commits/79b475bd9ffb7c389d9be7629bcf788d39329fd4

Reply:
I'll look at fixing the tests 😊